src/main/java/lotto/LottoNumber.java

```diff
@@ -1,16 +1,29 @@
 package lotto;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+
 public class LottoNumber {
     private final int number;
-
     private static final int LOTTO_NUMBER_MIN = 1;
     private static final int LOTTO_NUMBER_MAX = 45;
+    private static final Map<Integer, LottoNumber> LOTTO_NUMBER_POOL = new HashMap<>();
+
+    static {
+         IntStream.range(LOTTO_NUMBER_MIN, LOTTO_NUMBER_MAX + 1)
```

Comment:
`rangeClosed`를 활용해보면 어떨까요?
```suggestion
         IntStream.rangeClosed(LOTTO_NUMBER_MIN, LOTTO_NUMBER_MAX)
```